packages/clay-css/src/scss/components/_dropdowns.scss

```diff
@@ -93,6 +93,11 @@
 	}
 }
 
+html[dir="rtl"] .dropdown-menu {
+	left: auto;
+	right: 0;
+}
+
```

Comment:
@bryceosterhaus I believe portal's rtl css will swap these styles to ` left: 0; right: auto;`. I'm not sure if the library we use checks to see if the selector is `rtl` specific.

Reply:
Hey @pat270, I think that's exactly what @bryceosterhaus expects to override the current `right: 0` with a more specific `right: auto`.

This is pretty obscure, and I'm pretty ashamed of https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-css/frontend-css-web/src/main/resources/META-INF/resources/portal/_dropdown.scss#L5 myself. 

This seems to depend on a build process that happens in DXP outside of Clay... for a problem that might be self-inflicted by DXP... I would probably apply it in DXP first, but it's totally up to you.